group_vars/all/main.yml

```diff
@@ -11,6 +11,10 @@ www_root: /srv/www
 ip_whitelist:
   - "{{ (env == 'development') | ternary(ansible_default_ipv4.gateway, ipify_public_ip | default('')) }}"
 
+# Deploy using git archive to bundle up the repo, if using git submodules, set this to false
+deploy_using_git_archive: true
```

Comment:
I'd prefer a `deploy_method` variable with two options: `archive` and `copy` (or a better name).